packages/@react-aria/numberfield/src/useNumberField.ts

```diff
@@ -384,13 +390,18 @@ export function useNumberField(props: AriaNumberFieldProps, state: NumberFieldSt
 }
 
 // scroll wheel needs to be added not passively so it's cancelable, small helper hook to remember that
-function useScrollWheel({onScroll, capture}: {onScroll: (e) => void, capture: boolean}, ref: RefObject<HTMLElement>) {
+function useScrollWheel({onScroll, capture, disable}: {onScroll: (e) => void, capture: boolean, disable: boolean}, ref: RefObject<HTMLElement>) {
```

Comment:
Should be `isDisabled`

Reply:
yeah, this is addressed in https://github.com/adobe/react-spectrum/pull/1654/files#diff-cce470e90db62eae114c603721371a2c5d91e0007b4d7fe4b1ec0d100226fb6bR22

Reply:
eh, i'll have merge conflicts anyways, addressed here as well